migrate/20240701_add_firewall_location.rb

```diff
@@ -0,0 +1,9 @@
+# frozen_string_literal: true
+
+Sequel.migration do
+  change do
+    alter_table(:firewall) do
+      add_column :location, :text, null: false
```

Comment:
Does it work with existing firewalls? IMHO, this migration should fail since the new column is `null: false` and the existing ones don't have a location